fury/tests/test_material.py

```diff
@@ -504,7 +504,7 @@ def test_manifest_standard(interactive=False):
         sphere = get_sphere('symmetric362')
         shape = (11, 11, 11, sphere.vertices.shape[0])
         fid, fname = mkstemp(suffix='_odf_slicer.mmap')
-        odfs = np.memmap(fname, dtype=np.float64, mode='w+', shape=shape)
+        odfs = np.memmap(fname, dtype=float, mode='w+', shape=shape)
```

Comment:
same comment as above.
Can you revert this change?  `float` is platform-dependent. here, they are forcing float64 for whatever reason.